src/App.js

```diff
@@ -1,12 +1,65 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
+import axios from 'axios'
+// import pokemonLogo from './assets/pokemonLogo.svg'
+// Als ik het logo probeer weer te geven krijg ik een foutmelding die ik niet begrijp, dus om de applicatie te kunnen runnen, heb ik dit maar uitgezet.
+import Button from './components/button/Button';
+import PokemonCard from './components/pokemon-card/PokemonCard';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState([]);
+    const [error, setError] = useState(false);
+    const [loading, toggleLoading] = useState(false);
+    const [endpoint, setEndpoint] = useState('https://pokeapi.co/api/v2/pokemon/');
+
+    useEffect( () => {
+        async function fetchAllPokemons() {
+            toggleLoading(true);
+            setError(false);
+
+            try {
+                const response = await axios.get(endpoint);
+                setPokemons(response.data.results);
```

Comment:
Je haalt hier alleen Response.data.results binnen. Hiermee gooi je de data zoals previous en next in de prullen back.

Vervolgens vraag je dus 2 undefineds op. die zijn false. Maar omdat je !undefined doet wordt het true en zijn je buttons dus disabled.

Reply:
Ik denk dat ik snap wat je bedoelt, maar hoe moet ik het dan wel doen? Als ik probeer om alleen response.data binnen te halen, dan lukt het niet meer met mappen, ik krijg dan de melding "pokemons.map is not a function".

Reply:
Als je de hele response.data binnenhaalt zal je voor de dingen die met results te maken hebt ook weer de result key aan moeten dus bijvoorbeeld.

pokemons.results.map